src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -1177,7 +1177,10 @@ public bool TryGetAutoIndex(string name,out AutoIndexDefinition definition)
                 var deletionInProgress = new Dictionary<string, DeletionInProgressStatus>();
                 foreach (var prop in obj.GetPropertyNames())
                 {
-                    deletionInProgress[prop] = (DeletionInProgressStatus)obj[prop];
+                    if (Enum.TryParse(obj[prop].ToString(), out DeletionInProgressStatus result))
```

Comment:
Is this always going to be a string here?

Reply:
Something should go very wrong if this isn't the case.

Reply:
Yes, this is from:
```
public Dictionary<string, DeletionInProgressStatus> DeletionInProgress;
```

Reply:
@karmeli87 

```
obj[prop]?.ToString()
```

Maybe just in case change to that?